src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] To fix the issue, remove the line `/* eslint-disable no-unused-vars */` from your code. It's best practice to address unused variables or use the _ prefix to indicate intentionally unused variables, rather than disabling the ESLint rule altogether. Disabling rules should be done sparingly and with good reason, as it's generally better to improve code quality by resolving the underlying issues.
- [ ] You might want to check the [Documetation](https://eslint.org/docs/latest/rules/no-unused-vars) to fix this 👍